.changeset/smooth-worms-end.md

```diff
@@ -0,0 +1,5 @@
+---
+"@nomiclabs/hardhat-ethers": patch
```

Comment:
Given that this is changing the type signatures (and so can break existing code) should this be `minor`?

Reply:
We have opted to add functions rather alter existing signatures. Hence this can be reduced to minor update.